core/modules/qdisp/QueryRequest.cc

```diff
@@ -66,41 +67,43 @@ class QueryRequest::AskForResponseDataCmd : public PriorityCommand {
     void action(util::CmdData *data) override {
         // If everything is ok, call GetResponseData to have XrdSsi ask the worker for the data.
         {
+            util::InstanceCount _ic{"1-act-AskForResponseDataCmd"};
```

Comment:
Why this local variable starts with underscore? I think this naming convention is reserved for the non-public member variables of classes and structs.